linera-execution/src/execution_state_actor.rs

```diff
@@ -8,15 +8,48 @@ use crate::{
     UserApplicationDescription, UserApplicationId, UserContractCode, UserServiceCode,
 };
 use futures::channel::mpsc;
-use linera_base::data_types::{Amount, Timestamp};
+use linera_base::{
+    data_types::{Amount, Timestamp},
+    metrics::MeasureLatency,
+    sync::Lazy,
+};
 use linera_views::{
     batch::Batch,
     common::Context,
     views::{View, ViewError},
 };
 use oneshot::Sender;
+use prometheus::{register_histogram_vec, HistogramVec};
 use std::fmt::{self, Debug, Formatter};
 
+/// Histogram of the latency to load a contract bytecode.
+pub static LOAD_CONTRACT_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "load_contract_latency",
+        "Load contract latency",
+        &[],
+        vec![
+            0.001, 0.002_5, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0, 25.0,
+            100.0, 250.0,
+        ],
+    )
+    .expect("Histogram creation should not fail")
+});
+
+/// Histogram of the latency to load a service bytecode.
+pub static LOAD_SERVICE_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
Per @MathieuDutSik 's PR #1529, I believe these don't need to be `pub`.

Reply:
Oops, thanks for the heads-up! Fixed.